### PR TITLE
[63] Style the Contact Us Page using CSS

### PR DESCRIPTION
**Pull request by Marienoir**
##Task: Style the Contact Us Page using CSS
- `I styled the contact page using CSS`
phone1
phone2
phone3
tablet
tablet2
web
web1

### CODE REVIEW

**Review by ToluAkin — CHANGES_REQUESTED** [review]
@Marienoir You used a lot of width and height which will make your webpage pretty much static. If you want space around your elements use padding or margin. Try and take time to review page and use more of  padding or margin.

File: submissions/maryOkosun/css/contact.css [hunk 1]
```diff
@@ -0,0 +1,325 @@
+*{
+	box-sizing: border-box;
+	font-family: 'Montserrat', sans-serif;
+	
+}
+html,body{
+	box-sizing: border-box;
```
[the remark this answers is not in the document]
Comment (Marienoir) [reply submissions/maryOkosun/css/contact.css]: I have corrected that. Thanks

File: submissions/maryOkosun/css/contact.css [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+	font-size: 16px;
+}
+header img{
+	height: 100px;
+	float: left;
+	padding-bottom: 20px;
+	margin-top: 0;
+	margin-left: 30px;
+}
+body{
```
[the remark this answers is not in the document]
Comment (Marienoir) [reply submissions/maryOkosun/css/contact.css]: Done also

File: submissions/maryOkosun/css/contact.css [hunk 3]
```diff
@@ -0,0 +1,325 @@
...
+	padding: 0;
+	background-color: #24305E;
+	text-align: right;
+}
+nav li{
+	display: inline-block;
+	padding: 20px;
+}
+nav a{
+	height: 40%;
```
[the remark this answers is not in the document]
Comment (Marienoir) [reply submissions/maryOkosun/css/contact.css]: Thanks a lot. I have effected the change.

File: submissions/maryOkosun/css/contact.css [hunk 4]
```diff
@@ -0,0 +1,325 @@
...
+	padding: 5px ;
+	margin: 5px;
+	width: 100%;
+}
+main button{
+	display: inline-block;
+	text-align: center;
+	margin-top:15px;
+	margin-left: 320px;
+	width: 130px;
```
[the remark this answers is not in the document]
Comment (Marienoir) [reply submissions/maryOkosun/css/contact.css:99]: Okay. I used just width now in my new PR. Kindly review, thanks.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**